src/browser_action/vitals.js

```diff
@@ -224,17 +233,66 @@
     );
   }
 
-  function addUserTimings(metric) {
+  function addUserTimings(metric, enableUserTiming, enableConsoleTables) {
     switch (metric.name) {
       case "LCP":
```

Comment:
Since we're reporting "reportAllChanges" for these metrics, that means we'll log and create user timing for all metric candidates.  That's mostly a good thing, except maybe for LCP...

How does this look in DevTools when you load a page with multiple candidates reported over time?  Wont the timings overlap (TTFB etc) since all are reporting from 0?

Is there any way to know which when LCP is finalized with web-vitals.js?  We could console.log() all but user timings only for final?

Reply:
We're not using `reportAllChanges`. And I question the value of doing that since it's really "reportAllChangesThatAreBiggerThanTheLast", so still only partially useful to report all layout shifts (CLS) or interactions (for INP).

The only thing that it would be good for, would be reporting LCP right away (i.e. before a click) and same for CLS and INP (the extension currently requires you to tab away and back before it reports these). But if we did ever introduce that, then yes we'd need to rethink how we did this. But, as I say, I question it's value.

Reply:
Fixed in https://github.com/GoogleChrome/web-vitals-extension/pull/115/commits/991ff048d6947e9c40d000880abbd1761566a424

I think reporting multiple LCPs in this case is find. Not ideal but fine. It's already what happens with the existing console logging.